### ISSUE TITLE
Incomplete response received from application error when serving assets in Muscat 6.0

### ISSUE THREAD

**Issue by fjorba** [rism-ch/muscat#873]
I've spent much time trying to understand this issue, and I confess that I cannot solve it, so I think I'd better ask.

In the same server where I've installed three different Muscat 5.1.2 and 5.2.1 instances, previously with Debian 9 (Stretch) and now with Debian 10 (Buster), Muscat 6.0 don't show css and other static files. I've followed the same instructions twice for one account and once for another.

When trying to manually retrieve most of the css files (for example, /assets/blacklight.self-ca71897feb9ca9c296dd0e361edcd0e2345cbd51316110a7bd1d9d17789aafc0.css?body=1), I'm receiving this error, both in the browser and in the command line using curl:

```
<h2>Incomplete response received from application</h2>
```

None of the solutions I've tried googling around worked.

**Comment by HirschSt** [comment]
maybe running
```bash
bundle exec rake RAILS_ENV=production assets:clean
bundle exec rake RAILS_ENV=production assets:precompile
```
helps?

**Comment by fjorba** [comment]
Thanks for the suggestion, but I tried them, several times, in different accounts, and they didn't help, sorry.

**Comment by HirschSt** [comment]
probably related to #883

**Comment by fjorba** [comment]
Great!  This issue is solved in 6.1.  Thanks!
